python/tk_multi_loader/dialog.py

```diff
@@ -1842,8 +1844,11 @@ def _load_publishes_for_entity_item(self, item):
 
         if show_sub_items:
             # indicate this with a special background color
+            color = self.palette().highlight().color()
             self.ui.publish_view.setStyleSheet(
-                "#publish_view { background-color: rgba(44, 147, 226, 20%); }"
+                "#publish_view { background-color: rgba({red}, {green}, {blue}, 20%); }".format(
```

Comment:
Since you're using `format` you'll need to escape/double up the actual braces here:

#publish_view **{{** background-color: rgba({red}, {green}, {blue}, 20%); **}}**

Reply:
Right -- sorry about all these!